tools/common.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright 2019 Google Inc.
+# Copyright 2021 Google Inc.
```

Comment:
FYI You don't need to update copyright year of an existing file. (I guess you can, but you don't need to.) This is only necessary for new files.

Reply:
Thanks. I will revert it just in case.